services/drupal/web/themes/epa_theme/source/_patterns/05-components/box/_box.scss

```diff
@@ -22,6 +22,18 @@ $image-header-width: 320px;
   &.has-image-header {
     max-width: rem($image-header-width);
   }
+
+  .l-sidebar__sidebar &,
+  .l-grid &,
+  .l-resource-directory & {
+    &.u-align-left,
+    &.u-align-right {
+      float: none;
+      margin-left: 0;
+      margin-right: 0;
+      width: 100%;
+    }
+  }
```

Comment:
@jbwagner Instead of just targeting the `.box` component, we want to globally apply this fix to the `.u-align-left/right` classes.

Reply:
Fixed in commit 454e2e0